### PR TITLE
Fix references to VBOs causing disposal to never occur

### PR DESCRIPTION
**Pull request by peppy**
Eventually bottlenecks draw thread performance over long executions with many BufferedContainers being constructed.

### CODE REVIEW

File: osu.Framework/Graphics/OpenGL/GLWrapper.cs [hunk 1]
```diff
@@ -87,8 +88,7 @@ internal static void Reset(Vector2 size)
             lastBlendingInfo = new BlendingInfo();
             lastBlendingEnabledState = null;
 
-            foreach (IVertexBatch b in all_batches)
-                b.ResetCounters();
+            all_batches.ForEachAlive(b => b.ResetCounters());
```
Comment (smoogipoo) [inline osu.Framework/Graphics/OpenGL/GLWrapper.cs]: Delegate allocation every frame.

Comment (peppy) [reply osu.Framework/Graphics/OpenGL/GLWrapper.cs]: shouldn't be, this has no captures (and i'm 99% sure i tested this).

Comment (peppy) [reply osu.Framework/Graphics/OpenGL/GLWrapper.cs]: Re-confirmed via profiling this is not an issue.

Comment (smoogipoo) [reply osu.Framework/Graphics/OpenGL/GLWrapper.cs]: Seems ok.

**Pull request merged**